lib/ex_doc/language.ex

```diff
@@ -102,6 +115,8 @@ defmodule ExDoc.Language do
 
     * `:line` - the line where the code is located
 
+    * `:source` - the source file where the code is located
```

Comment:
Should we make these consistent? In some cases we have line+source, in others we have source_line+source_file? Or are they different on purpose?

Reply:
yes. I'll fix that.

Reply:
The last work is to keep it as `line`+`source` everywhere, but it is your call!

Reply:
just realised that it is already called source_line and source_file everywhere, it was just the callback specs that were not updated. Will fix the callback definitions.